docs/docusaurus/docs/reference/learn/expectations/distributional_expectations.md

```diff
@@ -21,7 +21,7 @@ continuous data or categorical classes and their associated weights.
 
 * A partition is defined by an ordered list of points that define intervals on the real number line. Note that partition
   intervals do not need to be uniform.
-* Each bin in a partition is partially open: a data element x is in bin i if lower_bound_i <= x < upper_bound_i.
+* Each bin in a partition is partially open: a data element x is in bin i if lower_bound_i &lt;= x &gt; upper_bound_i.
```

Comment:
Wouldn't this second one also be `&lt;`?

Reply:
@JessSaavedra, yes, it should. This is what happens when you review almost 300 files! It has been corrected. Good catch!